codec-http2/src/main/java/io/netty/handler/codec/http2/Http2RemoteFlowController.java

```diff
@@ -27,8 +27,8 @@
      * guarantee when the data will be written or whether it will be split into multiple frames
      * before sending.
      * <p>
-     * Manually flushing the {@link ChannelHandlerContext} is not required, since the flow
-     * controller will flush as appropriate.
+     * Manually flushing the {@link ChannelHandlerContext} is required for writes as the flow controller will
+     * not flush by itself.
```

Comment:
Update the comment on line [85](https://github.com/netty/netty/pull/3712/files#diff-3ad0811d613c730f2815be14880147fdR85) to remove the `flush is required`.